tests/browser/js/simplificationTests.js

```diff
@@ -202,10 +202,33 @@
                         listener: "gpii.tests.simplifyTests.assertNotSimplified",
                         args: ["{simplify}"]
                     }]
+                }, {
+                    name: "Dynamically added always visible element",
+                    expect: 1,
+                    sequence: [{
+                        func: "{simplify}.applier.change",
+                        args: ["simplify", true]
+                    }, {
+                        func: "fluid.tests.simplifyTester.injectElm",
+                        args: ["{simplify}"]
+                    }, {
+                        event: "{simplify}.events.onAlwaysVisibleNodeAdded",
+                        priority: "after:makeVisible",
+                        listener: "fluid.tests.simplifyTester.assertElmVisible"
+                    }]
                 }]
             }]
         });
 
+        fluid.tests.simplifyTester.injectElm = function (that) {
```

Comment:
I don't think the abbreviation "Elm" for "Element" aids readability too much given the reader's first thought might be of trees : P